core/quantity/src/Formatter/FormatEnums.ts

```diff
@@ -112,6 +112,8 @@ export enum FormatType {
   Bearing,
   /** Azimuth angle e.g. 45°30'00". Requires provided quantities to be of the angle phenomenon */
   Azimuth,
+  /** Eg. 1:2 or 0.3:1. Requires provided quantities to be of the slope phenomenon */
```

Comment:
is slope required?  Is something invertible required if we don't do inversion?  Would miles per gallon work so long as we didn't ask for gallons per mile?

Reply:
yeah it's not required that was a doc oversight. Naron has deleted the requirement.